src/dialog/an-dialog.factory.js

```diff
@@ -30,9 +30,9 @@
                         '   </div>',
                         '</div>'
                     ].join('\n');
-    
-                    var getBlanket = function() {
-                        return $body.find('.an-dialog-blanket');
+
+                    var getBlanketsArray = function() {
+                        return Array.prototype.slice.call(document.querySelectorAll('.an-dialog-blanket'));
```

Comment:
This is very fragile, because you are relying on the browser to enumerate the matches for the selector in the same sequence that the dialogs are stacked.
The blankets should be maintained in the same way we maintain the dialog stack.